test/array.js

```diff
@@ -1,20 +1,18 @@
 var morse = require('../');
+var assert = require('assert');
 
-var arr = [
-  'hello',
-  'world'
-];
-console.log(arr);
+describe('array', function() {
+	it('should encode and decode an array', function() {
+		var arr = [
+		  'hello',
+		  'world'
+		];
 
-var encoded = morse.encode(arr);
-console.log(encoded);
+		var encoded = morse.encode(arr);
+		var decoded = morse.decode(encoded);
 
-var decoded = morse.decode(encoded);
-console.log(decoded);
-
-var expected = [
-  'HELLO',
-  'WORLD'
-];
-
-console.log(expected[1] == decoded[1] ? 'pass' : 'fail');
+		assert.deepEqual(decoded, arr.map(function(str) {
+			return str.toUpperCase();
+		}), 'Arrays should be equal except for case');
+	});
```

Comment:
Oh sorry, my intention was not to change that file, sorry. I was just
playing with mocha

2015-01-31 20:13 GMT+01:00 Cam Pedersen <notifications@github.com>:

> In test/array.js
> <https://github.com/ecto/morse/pull/3#discussion_r23891251>:
>
> >
> > -var decoded = morse.decode(encoded);
> > -console.log(decoded);
> > -
> > -var expected = [
> > -  'HELLO',
> > -  'WORLD'
> > -];
> > -
> > -console.log(expected[1] == decoded[1] ? 'pass' : 'fail');
> > +		assert.deepEqual(decoded, arr.map(function(str) {
> > +			return str.toUpperCase();
> > +		}), 'Arrays should be equal except for case');
> > +	});
>
> Still some tabs in test/array.js
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ecto/morse/pull/3/files#r23891251>.
>